vuepress/docs/next/docs/ecr/ecr-bundle-versions-faq.md

```diff
@@ -0,0 +1,73 @@
+# Bundle versions and updates - FAQ
+
+## Does the ECR support versioning?
+A bundle, as already discussed in the [overview](./ecr-overview.md), is a package containing one or more components.
```

Comment:
remove "already" (we don't know if they read that yet or not). Should read

`A bundle, as discsussed in`